### DISCUSSION TITLE
v-ripple Cannot read properties of undefined (reading 'config')

### DISCUSSION THREAD

**Discussion by lucho19jose** [quasarframework/quasar#13661] [General - Components / Directives / etc] [answered]
Hi, I got this error when I upgrade @quasar/app-vite project to v1.0.1 and quasar 2.7.1. why is it happened?
![askforhelp](https://user-images.githubusercontent.com/46167630/172921609-7a109c1f-20db-4f79-9aeb-ada548aa049b.PNG)

**Comment by suleimanas** [comment, marked as answer]
it's a bug in Vue
https://github.com/quasarframework/quasar/issues/13154

**Comment by lucho19jose** [comment]
then I have to remove v-ripple from components?

**Reply by suleimanas** [reply]
yes, for now

**Reply by suleimanas** [reply]
or you can downgrade Vue to 3.2.31
